lib/resolver.content.js

```diff
@@ -133,13 +134,29 @@ export default class PodletClientContentResolver {
             outgoing.contentUri,
         );
 
+        let assetObjects;
+
         /** @type {import('./http.js').PodiumHttpClientRequestOptions} */
         const reqOptions = {
             rejectUnauthorized: outgoing.rejectUnauthorized,
             timeout: outgoing.timeout,
             method: 'GET',
             query: outgoing.reqOptions.query,
             headers,
+            onInfo: ({ statusCode, headers }) => {
+                if (statusCode === 103) {
+                    // send early hints
+                    assetObjects = utils.parseLinkHeaders(headers.link);
+                    if (assetObjects) {
+                        const preloads =
+                            utils.toPreloadAssetObjects(assetObjects);
+                        const link = preloads.map((preload) =>
+                            preload.toHeader(),
+                        );
+                        outgoing.writeEarlyHints({ link });
+                    }
+                }
+            },
```

Comment:
We send off early hints as soon as we get early hints from the podlet. The podlet link header has to be parsed, then converted into a preload link format for the browser.